packages/cadl-rlc-emitter/src/modelUtils.ts

```diff
@@ -548,10 +549,7 @@ function applyIntrinsicDecorators(
     const values = getKnownValues(program, cadlType);
     if (values) {
       const enumSchema = { ...newTarget, ...getSchemaForEnum(program, values) };
-      enumSchema["x-ms-enum"].modelAsString = true;
```

Comment:
I'd like to confirm that the `x-ms-enum` extension will NOT appear in CADL model, Right?

I think this is only useful for swagger input.

Reply:
yes, we don't need this.